src/EnergyPlus/HVACDXHeatPumpSystem.cc

```diff
@@ -523,12 +518,12 @@ namespace HVACDXHeatPumpSystem {
         // These initializations are done every iteration
         if (AirLoopNum == -1) { // This IF-Then routine is just for ZoneHVAC:OUTDOORAIRUNIT
 
-            DXHeatPumpSystem(DXSystemNum).DesiredOutletTemp = OAUCoilOutletTemp;
+            state.dataHVACDXHeatPumpSys->DXHeatPumpSystem(DXSystemNum).DesiredOutletTemp = OAUCoilOutletTemp;
 
-        } else if (AirLoopNum != -1) { // Not Outdoor Air Unit
-            ControlNode = DXHeatPumpSystem(DXSystemNum).DXSystemControlNodeNum;
```

Comment:
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:528]:(style),[multiCondition],Expression is always true because 'else if' condition is opposite to previous condition at line 524.